src/screens/PastEvents/Admin/EventManager.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { Button } from "reactstrap";
+import Icon from "../../../components/Icon";
+import EventTable from "./EventTable";
+import { fetchEvents } from "../../../actions/queries";
+import EventCreateModal from "./EventCreateModal";
+import EventEditModal from "./EventEditModal";
+import EventDeleteModal from "./EventDeleteModal";
```

Comment:
Can remove these as we wont require to create, edit or delete event from the past-events page.